nannyml/plots/util.py

```diff
@@ -15,7 +15,7 @@
 from nannyml.exceptions import InvalidArgumentsException
 
 
-def is_time_series(time_series: Optional[Union[np.ndarray, pd.Series]]) -> TypeGuard[Union[np.ndarray, pd.Series]]:
+def has_non_null_data(time_series: Optional[Union[np.ndarray, pd.Series]]) -> TypeGuard[Union[np.ndarray, pd.Series]]:
```

Comment:
Not sure about the name of the `time_series` parameter here? Maybe just `data` or `series`?

Reply:
Good point! Missed that in generalising the function. `series` seems good as both the numpy and pandas version are one-dimensional